src/components/ModalEmailHOC.vue

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
```

Comment:
What does HOC mean, can you use a more descriptive name?

Reply:
You only use it in one place, so I don't really see the need for the HOC component

Reply:
ok, I see. Will remove it then.